### ISSUE TITLE
Emoji in string raises error

### ISSUE THREAD

**Issue by Jip-Hop** [baalexander/node-xmlrpc#122]
Hi,

I'm trying to submit a string that contains emoji's. It raises the following error:
```
Error: Invalid character (�) in string: 👌🏻 Thumbs up 👍🏻for the writers
    at XMLFragment.assertLegalChar (/node_modules/wordpress/node_modules/xmlrpc/node_modules/xmlbuilder/lib/XMLFragment.js:322:15)
    at XMLFragment.text (/node_modules/wordpress/node_modules/xmlrpc/node_modules/xmlbuilder/lib/XMLFragment.js:137:12)
    at XMLFragment.txt (/node_modules/wordpress/node_modules/xmlrpc/node_modules/xmlbuilder/lib/XMLFragment.js:334:19)
    at appendString (/node_modules/wordpress/node_modules/xmlrpc/lib/serializer.js:175:23)
    at serializeValue (/node_modules/wordpress/node_modules/xmlrpc/lib/serializer.js:92:11)
    at /node_modules/wordpress/node_modules/xmlrpc/lib/serializer.js:25:5
    at Array.forEach (native)
    at Object.exports.serializeMethodCall (/node_modules/wordpress/node_modules/xmlrpc/lib/serializer.js:24:10)
    at Client.methodCall (/node_modules/wordpress/node_modules/xmlrpc/lib/client.js:76:30)
    at Client.extend.call (/node_modules/wordpress/lib/wordpress.js:68:12)
```
I think this problem may be related to this issue: https://github.com/oozcitak/xmlbuilder-js/issues/98

**Comment by g-gabber** [comment]
i faced the same problem :)

**Comment by patricklodder** [comment]
Fixed with #127, will be released with v1.3.2 this week
